AsyncDisplayKit/Private/_ASPendingState.mm

```diff
@@ -660,144 +667,175 @@ - (void)applyToView:(UIView *)view
 
   CALayer *layer = view.layer;
 
-  if (_flags.setAnchorPoint)
+  ASPendingStateFlags flags = _flags;
+  if (flags.needsDisplay
+      || (flags.setOpaque && opaque != view.opaque)
+      || (flags.setBackgroundColor && !CGColorEqualToColor(backgroundColor, layer.backgroundColor))) {
```

Comment:
It seems like code of this nature should not appear in two places. Maybe we can use a #define  macro, or an in-line function?